tools/challenge-parser/parser/plugins/utils/get-section.js

```diff
@@ -4,27 +4,26 @@ const findAllAfter = require('unist-util-find-all-after');
 const between = require('unist-util-find-all-between');
 const { findAll } = require('./find-all');
 
+const isMarker = node => {
+  if (node.children && node.children[0]) {
+    const child = node.children[0];
+    return (
+      child.type === 'text' &&
+      child.value.startsWith('--') &&
+      child.value.endsWith('--') &&
+      node.type === 'heading'
+    );
+  } else {
+    return false;
+  }
```

Comment:
```suggestion
  if (node.children && node.children[0]) {
    const child = node.children[0];
    return (
      child.type === 'text' &&
      child.value.startsWith('--') &&
      child.value.endsWith('--') &&
      node.type === 'heading'
    );
  };
  return false;
```

`if` statements which end in a return do not need `else`.